### ISSUE TITLE
Support for multiple UV channels

### ISSUE THREAD

**Issue by DavidArayan** [kcoley/gltf2usd#149]
Hello,

We have been using this tool and it's absolutely fantastic, so thank you for sharing this tool!

I'd like to ask if it were possible to add support for multi UV channels? For example, currently trying to export GLTF with baked lightmaps (in seperate uv channels) looks messed up. Does USDZ actually support multiple UV channels?

Have a great day!

**Comment by kcoley** [comment]
Hi @DavidArayan .  I just merged a fix to support the `TEXCOORD_1` attribute, so two UV channels should be supported.  Are you trying to use more than two UV maps?

**Comment by DavidArayan** [comment]
Hello @kcoley . We are just using 2 UV channels at the moment, 2nd being for lightmaps. That was really quick work thank you! will pull, and try it out.

**Comment by DavidArayan** [comment]
Hello @kcoley Sorry, just tried out your changes but the lightmaps in the secondary UV channel are still not applied to the USDZ model. I'll try to source you an example GLTF file to check out.

**Comment by DavidArayan** [comment]
Hello @kcoley 

I've got the files for reference.

USDZ
<img width="589" alt="screen shot 2019-02-28 at 5 50 18 pm" src="https://user-images.githubusercontent.com/7587896/53546854-8df56380-3b81-11e9-9563-b8d28f70b87e.png">

GLTF
<img width="567" alt="screen shot 2019-02-28 at 5 50 28 pm" src="https://user-images.githubusercontent.com/7587896/53546861-93eb4480-3b81-11e9-94d5-beb5b416699d.png">

Original files (as a zip) contains the .GLTF file and the converted .USDZ file
[UV_Cube.zip](https://github.com/kcoley/gltf2usd/files/2913729/UV_Cube.zip)

**Comment by kcoley** [comment]
@DavidArayan thanks for the sample model.  It looks like the occlusion texture is actually set in the usdz file, but QuickLook seems to be multiplying the occlusion map to the diffuse color.  However, if you view the usdz file in Xcode, it seems to behave more like ThreeJS, where the occlusion texture effect is "decreased" when facing a light source:
<img width="500" alt="screen shot 2019-02-28 at 6 04 09 am" src="https://user-images.githubusercontent.com/3946928/53571853-b8fa9b80-3b1e-11e9-9ea2-55a1b5620c6a.png">

However, the UV map for Three.js used for occlusion seems to be incorrect.

The usdz file looks similar to how BabylonJS is rendering the model in the sandbox:
<img width="428" alt="screen shot 2019-02-28 at 6 05 42 am" src="https://user-images.githubusercontent.com/3946928/53571929-ee06ee00-3b1e-11e9-850f-b3a405a74313.png">

But again, it looks like the occlusion is baked with the diffuse color.

So in summary, it seems to be an issue with how Quicklook is handling AO.

**Comment by DavidArayan** [comment]
Hello @kcoley ,

Thank you for your insight. I might end up writing an external tool that either merges the UV channels into a single channel or just not use lightmaps.

**Comment by kcoley** [comment]
@DavidArayan no problem, and good luck!
